File: python/my_stack.py
from queue import LifoQueue

stack = LifoQueue()
stack.put(1)
stack.put(2)
stack.put(3)
stack.put(3)

print("Size of stack is %d." % stack.qsize())
print("Top element of the stack is %d." % stack.get())

stack.put(4)
stack.put(5)
print("LIFO: ", end="")
while not stack.empty():
    print(stack.get(), end=" ")

File: cpp/my_stack.cpp
#include<bits/stdc++.h>
using namespace std;


int main(){
    stack<int> s;
    s.push(1);
    s.push(2);
    s.push(3);
    cout<<s.top()<<endl;
    s.pop();
    cout<<s.top()<<endl;
    s.push(4);
    s.push(5);
    cout<<"Size of stack is: "<<s.size()<<endl;
    cout<<s.empty()<<endl;


    return 0;
}
